### PR TITLE
ITS HIP TO GOOF BEES

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
add: GoofBees(tm) have arrived on the station.
experiment: Collect all 6 variants now!
tweak: Bees now have slightly more base health.
fix: Bees no longer uselessly sting silicons.
/:cl:

Need sprites for the bee variants.

TODO:
- [ ] More bee types
- [ ] Bees having more than one reagent
- [ ] New Frames

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/bees.dm [hunk 1]
```diff
@@ -48,7 +48,46 @@
...
+
+/mob/living/simple_animal/hostile/poison/bees/vapor//wave
+	name = "vaporizing bee"
+	desc = "You don't understand, it's all in your head"
+	transfer_type = VAPOR//WAVE
+
+/mob/living/simple_animal/hostile/poison/bees/touch
+	name = "fuzzy bee"
+	desc = "You think touching that is a bad idea."
+	transfer_type = TOUCH
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: styptic powder beeees

**Comment by PKPenguin321** [comment]
how do you get them

**Comment by Fox-McCloud** [comment]
It picks subtypes when bees spawn

**Comment by Bawhoppen** [comment]
can you make all bees naturally want to try to collect reagents from stuff nearby if they have none?

File: code/modules/mob/living/simple_animal/hostile/bees.dm [hunk 2]
```diff
@@ -48,7 +48,46 @@
...
+	name = "high-capacity bee"
+	desc = "This bee has an engorged sack. Looks kinda weak, though."
+	maxHealth = 5
+	health = 5
+	transfer_min = 5
+	transfer_max = 10
+	
+/mob/living/simple_animal/hostile/poison/bees/strong
+	name = "big bee"
+	desc = "This bee has been lifting lately.."
```
Comment (GunHog) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: serveimage

Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/bees.dm]: whoever sprites this, I demand this bee the sprite in honor of the fallen goonswarm federation

**Comment by ma44** [comment]
Can we make the bees inject omnizine?

**Comment by Jazaen** [comment]
Yes, you can do that even with default ones. Remember, omnizine overdoses!

**Comment by bobdobbington** [comment]
How are Goofbees gonna interact with Remiebees? Do these guys use the same apiary/cargo crate/royal jelly system? Do they apply the same bonuses to plants?

**Comment by RemieRichards** [comment]
@bobdobbington Unusually for goof his changes here are good.
Goofbees are still Remiebees, goof's just doing things like:
* Letting some bees spawn with random features (More HP, Less HP, Melee damage, A different reagent injection method (VAPOR, INJECT, etc.))
* Adding upgraded versions of the frame objects (I set these objects up nicely for being expandable, things like "more bees per frame")
* Bees with multiple reagents

**Comment by Iamgoofball** [comment]
my bees are more hip though

**Comment by Firecage** [comment]
Mmmm. @Iamgoofball I do have a question though. Lets say you make a bee with 2 reagents, and those 2 reagents are used in a recipe, will they mix inside the bee, or will they mix inside the mob they are injected into.

**Comment by Iamgoofball** [comment]
inside the mob when injected

**Comment by KorPhaeron** [comment]
I'm gonna help goof stay focused on guncrafting

**Pull request closed without merging**